src/testsuite/vec_f128_dummy.c

```diff
@@ -261,8 +263,9 @@ _test_xfer_bin128_2_vui16t (__binary128 f128)
 
 #ifdef __FLOAT128__
 /* Mostly compiler and library tests follow to see what the various
- * compilers will do.  */
-
+ * compilers will do.  
+ * Need to disable __clang__ because it doesn't. */
```

Comment:
doesn't ... what?
Also, a blank line above this one would help separate the two statements, which are unrelated.

Reply:
#ifndef __clang__ // to disable __clang__ because <math.h> fails